scipy/special/cephes/owens_t.c

```diff
@@ -0,0 +1,283 @@
+    0.85266826283219451090E-01, 0.16245071730812277011E+00,
+    0.25851196049125434828E+00, 0.36807553840697533536E+00,
+    0.48501092905604697475E+00, 0.60277514152618576821E+00,
+    0.71477884217753226516E+00, 0.81475510988760098605E+00,
+    0.89711029755948965867E+00, 0.95723808085944261843E+00,
+    0.99178832974629703586E+00};
+
+static double WTS[] = {
+    0.18831438115323502887E-01, 0.18567086243977649478E-01,
+    0.18042093461223385584E-01, 0.17263829606398753364E-01,
+    0.16243219975989856730E-01, 0.14994592034116704829E-01,
+    0.13535474469662088392E-01, 0.11886351605820165233E-01,
+    0.10070377242777431897E-01, 0.81130545742299586629E-02,
+    0.60419009528470238773E-02, 0.38862217010742057883E-02,
+    0.16793031084546090448E-02};
+
+int get_method(double h, double a) {
+    int ihint = 14;
+    int iaint = 7;
+    int i = 0;
+
+    for (i = 0; i < 14; i++) {
+        if (h <= HRANGE[i]) {
+            ihint = i;
+            break;
+        }
+    }
+
+    for (i = 0; i < 7; i++) {
+        if (h <= HRANGE[i]) {
+            iaint = i;
+            break;
+        }
+    }
+
+    return SELECT_METHOD[iaint* 15 + ihint];
+}
+
+double owens_t_norm1(double x) {
+    return erf(x / sqrt(2)) / 2;
+}
+
+double owens_t_norm2(double x) {
+    return erfc(x / sqrt(2)) / 2;
+}
+
+double owensT1(double h, double a, double m) {
+
+    double aa = a * a;
+    double hh = -1 * h * h / 2;
+    double h_exp = exp(hh);
+
+    int i = 0;
+    int div = 1;
+    double aj = a / (2 * NPY_PI);
+    double dj = expm1(hh);
+    double gj = hh * h_exp;
+
+    double result = atan(a) * (1 / (2 * NPY_PI));
+
+    for (i = 0; i <= m; i++) {
+        result += dj * aj / div;
+        div += 2;
+        aj *= aa;
+        dj = gj - dj;
+        gj *= hh / i;
+    }
+
+    return result;
+}
+
+double owensT2(double h, double a, double m) {
+
+    double hh = h * h;
+    double ah = -0.5 * h * h * a * a;
+    double pi_sq = sqrt(2 * NPY_PI);
+
+    int i = 0;
+    double z = (ndtr(a * h) - 0.5) / h;
+
+    double result = 0;
+
+    for (i = 0; i <= (2 * m); i++) {
+        result += pow(-1, i - 1) * z;
+        z = ((2 * i - 1) * z - (pi_sq * pow(a, 2 * i - 1) *
+             exp(ah))) / (hh);
+    }
+
+    result *= -pi_sq * exp(-0.5 * hh);
+
+    return result;
+}
+
+double owensT3(double h, double a, double m) {
+    double aa = a * a;
+    double hh = h * h;
+    double y = 1 / hh;
+
+    int i = 0;
+    double vi = a * exp(-aa * hh / 2) / sqrt(2 * NPY_PI);
+    double zi = owens_t_norm1(a * h) / h;
+    double result = 0;
+
+    for(i = 0; i<= m; i++) {
+        result += zi * C[i];
+        zi = y * ((2 * i + 1) * zi - vi);
+        vi *= aa;
+    }
+
+    result *= exp(-hh / 2) / sqrt(2 * NPY_PI);
+
+    return result;
+}
+
+double owensT4(double h, double a, double m) {
+    double maxi = 2 * m + 1;
+    double hh = h * h;
+    double aa = -a * a;
+
+    int i = 1;
+    double ai = a * exp(-hh * (1 - aa) / 2) / (2 * NPY_PI);
```

Comment:
Could you change `aa` to something like `naa`? (And similar for some other variables in the code.)